fiftyone/utils/paddleocr_vl.py

```
@@ -0,0 +1,233 @@
+            config.name_or_path
+        )
+        model = transformers.AutoModelForImageTextToText.from_pretrained(
+            config.name_or_path, dtype=torch.bfloat16
+        ).eval()
+        return model.to(self._device)
+
+    @property
+    def media_type(self):
+        return "image"
+
+    def _predict_all(self, imgs):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_predict_all`

(ANN202)